### DISCUSSION TITLE
New tag not shown in Tags page

### DISCUSSION THREAD

**Discussion by CodeChoVui** [adityatelange/hugo-PaperMod#686] [Q&A] [answered]
I am trying to create new post with tags, but the tag did not show in the Tags page. Can anyone help me please.

This is my posts with tag `code`
![image](https://user-images.githubusercontent.com/85174661/147263507-88f6339f-7df8-4fcd-b05f-cf27fb57ad71.png)

But in the Tags page, it did not show up
![image](https://user-images.githubusercontent.com/85174661/147263626-afba6d85-c4fb-4d24-8795-46bfdfc63c21.png)

**Comment by svenbrier** [comment]
Please try:
`tags: ["First Post", "Hello World"]`

**Comment by Kunniii** [comment, marked as answer]
The problem might from Hugo. I just need to restart the live server and everything works fine. Thanks, anyway!
